data/de.haeckerfelix.gradio.style.css

```diff
@@ -8,3 +8,11 @@
 	font-size: smaller;
 	padding: 2px 2px;
 }
+
+.bitrate-text {
+	font-size: smaller;
```

Comment:
Where is this required?

Reply:
It was used for the text in the statusbar, I'll revert this.